Document/0x04a-Mobile-App-Taxonomy.md

```diff
@@ -44,16 +49,7 @@ Following is a non-exhaustive list of more popular frameworks for developing _hy
 - [Onsen UI](https://onsen.io/ "Onsen UI")
 - [Sencha Touch](https://www.sencha.com/products/touch/ "Sencha Touch")
 
-## Progressive Web App
-
-Progressive Web Apps (PWA) load like regular web pages, but differ from usual web apps in several ways. For example it's possible to work offline and access to mobile device hardware is possible, that traditionally is only available to native mobile apps.
-
-PWAs combine different open standards of the web offered by modern browsers to provide benefits of a rich mobile experience. A Web App Manifest, which is a simple JSON file, can be used to configure the behavior of the app after "installation".
-
-PWAs are supported by Android and iOS, but not all hardware features are yet available. For example Push Notifications, Face ID on iPhone X or ARKit for augmented reality is not available yet on iOS. An overview of PWA and supported features on each platform can be found in a [Medium article from Maximiliano Firtman](https://medium.com/@firt/progressive-web-apps-on-ios-are-here-d00430dee3a7 "Progressive Web Apps on iOS are here").
-
-## What's Covered in the Mobile Testing Guide
-
-Throughout this guide, we will focus on apps for Android and iOS running on smartphones. These platforms are currently dominating the market and also run on other device classes including tablets, smartwatches, smart TVs, automotive infotainment units, and other embedded systems. Even if these additional device classes are out of scope, you can still apply most of the knowledge and testing techniques described in this guide with some deviance depending on the target device.
+## <A id="progressive-web-apps">Progressive Web Apps</A>
 
-Given the vast amount of mobile app frameworks available it would be impossible to cover all of them exhaustively. Therefore, we focus on _native_ apps on each operating system. However, the same techniques are also useful when dealing with web or hybrid apps (ultimately, no matter the framework, every app is based on native components).
+_Progressive web apps_ (PWAs) combine different open standards of the web offered by modern browsers to provide benefits of a rich mobile experience. A Web App Manifest, which is a simple JSON file, can be used to configure the behavior of the app after "installation". These apps load like regular web pages, but differ from usual web apps in several ways.
+For example, it's possible to work offline and access to mobile device hardware is possible, which has been a capacity that was only available to _native apps_. PWAs are supported by both Android and iOS, but not all hardware features are yet available. For example, Push Notifications, Face ID on iPhone X, or ARKit for augmented reality is not available yet on iOS.
```

Comment:
Fix for 

Error: Document/0x04a-Mobile-App-Taxonomy.md:55:367 MD047/single-trailing-newline Files should end with a single newline character

```suggestion
For example, it's possible to work offline and access to mobile device hardware is possible, which has been a capacity that was only available to _native apps_. PWAs are supported by both Android and iOS, but not all hardware features are yet available. For example, Push Notifications, Face ID on iPhone X, or ARKit for augmented reality is not available yet on iOS.

```

Reply:
OK